### ISSUE TITLE
react-day-picker is misusing the application role

### ISSUE THREAD

**Issue by trezy** [gpbl/react-day-picker#548]
According to [the W3C](https://www.w3.org/WAI/GL/WCAG20-TECHS/ARIA11):

> ...the role of `application` should only be used with caution because it gives a signal to screen reading software to turn off normal web navigation controls. **Simple widgets should generally not be given the application role**, nor should an entire web page be given the application role, unless it is not to be used at all like a web page, and not without much user testing with assistive technology.)

However, `react-day-picker` [has the `application` role](https://github.com/gpbl/react-day-picker/blob/f7e5cd4a34af6f4cfe587a8f77c288b4d3a3e669/src/DayPicker.js#L516).

**Comment by gpbl** [comment]
@trezy thanks for pointing this out - which role should be used instead?

**Comment by trezy** [comment]
@gpbl, props for the a11y label and the awesome amount of ARIA implementation in this plugin! For this library, I think you're best just sticking with the `grid` role which is already set up. It's already a subclass of the `widget` role, which indicates specifically that it isn't an `application`. Just removing the `application` role would be sufficient to bring it back in line with ARIA.

**Comment by trezy** [comment]
@gpbl Thanks for doing that work! It's always awesome to see an OSS project doing so well. 😁

**Comment by gpbl** [comment]
You are welcome, thanks for the support. I published the change in v7
